src/vaadin-avatar.html

```diff
@@ -13,23 +13,32 @@
     <style>
       :host {
         display: inline-block;
+        border-radius: 50%;
+        overflow: hidden;
+        height: 44px;
+        width: 44px;
+      }
+
+      img {
+        height: 100%;
+        width: 100%;
       }
 
       :host([hidden]) {
         display: none !important;
       }
     </style>
-    <slot></slot>
+    <img hidden$="[[!__imgVisible]]" src$="[[img]]" />
```

Comment:
Do we need a property? Maybe we can change this to `hidden$="[[__computeImgHidden(img)]]"`.

Not a blocker in case we need this later, haven't checked the second PR yet.

Reply:
Ok, checked the second PR and it is used there. Marking as resolved.